app/controllers/converts_controller.rb

```diff
@@ -28,6 +28,9 @@ def create
       filename: file_name,
       layout: false
     )
+  rescue ActionController::ParameterMissing
+    flash.now[:alert] = "Missing 'file' or 'format' form option."
```

Comment:
Yeah, that makes more sense.  Trying to cover things that wouldn't normally happen without a user tinkering with the page is kind of silly in retrospect.